engine/occurrence.ftl

```diff
@@ -167,6 +208,19 @@
             }
         ) ]
 
+    [#-- define the setting namespaces for the component --]
+    [#local occurrence =
+        mergeObjects(
+            occurrence,
+            {
+                "Configuration" : {
+                    "SettingNamespaces" : internalGetOccurrenceSettingNamepsaces(occurrence)
```

Comment:
Type here in `Namespaces`, copied from the function definition I'd say

Reply:
Fixed